src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageService`IVsLanguageDebugInfo.cs

```diff
@@ -68,35 +78,27 @@ public int GetProximityExpressions(IVsTextBuffer pBuffer, int iLine, int iCol, i
                 return VSConstants.E_FAIL;
             }
 
-            var dialogResult = _waitDialogFactory.TryCreateWaitDialog(
+            IReadOnlyList<string> proximityExpressions = null;
+            var dialogResult = _uiThreadOperationExecutor.Execute(
                 title: "Determining proximity expressions...",
```

Comment:
Not localized?

Reply:
#razorlife

Reply:
We don't currently localize things in Razor because we didn't think we had any/much UI text. That's turned out to not be true in this and other cases, so I've filed https://github.com/dotnet/aspnetcore/issues/33084 to fix this.